cumulus/polkadot-parachain/src/command.rs

```diff
@@ -279,39 +276,9 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 		path => {
 			let path: PathBuf = path.into();
 			match path.runtime() {
```

Comment:
Also this match is useless?

Reply:
[a457c64](https://github.com/paritytech/polkadot-sdk/pull/2457/commits/a457c64a84962fe0e9419947053d9ea3c01b1241)